src/test/scala/com/microsoft/hyperspace/goldstandard/TPCDSBase.scala

```diff
@@ -554,17 +553,35 @@ trait TPCDSBase extends SparkFunSuite with SparkInvolvedSuite {
        """.stripMargin)
   }
 
+  private val originalCBCEnabled = conf.cboEnabled
```

Comment:
all changes below this line onward are directly picked from spark codebase. Refer https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/TPCDSBase.scala#L609